lib/game.rb

```diff
@@ -46,4 +46,11 @@ def initialize(game_stats)
     @venue_link = game_stats[:venue_link]
   end
 
+  def highest_total_score
+    @@all.map { |game| game.away_goals + game.home_goals}.max
+  end
+
+  def lowest_total_score
+    @@all.map { |game| game.away_goals + game.home_goals}.min
+  end
```

Comment:
does min_by work as well? my thoughts here are that you are mapping the current array by iterating through each item then you are calling .min on the array which is another enumerable so it might take a longer time to run

Reply:
seconded